arches_querysets/bulk_operations/tiles.py

```diff
@@ -214,6 +214,8 @@ def _update_tile(
                 if isinstance(container, TileModel):
                     new_tile.parenttile = container
                 new_tile.full_clean()
+                if new_tile.pk is None:
+                    new_tile.pk = uuid.uuid4()
```

Comment:
This was not strictly related to the fix, but wanted to add safety before I forgot.

This is only here just in case other paths (not over the API) manage to get child tiles scheduled for saving without having their pk set yet by the database. Alternatively the type of `to_insert` could be changed to a list rather than a set.